gtsfm/utils/triangulation.py

```diff
@@ -0,0 +1,41 @@
+"""Utilities to support triangulation.
+
+Authors: Ayush Baid
+"""
+import numpy as np
+from gtsam import PinholeCameraCal3Bundler
+
+
+def calculate_triangulation_angle_in_degrees(
+    camera_1: PinholeCameraCal3Bundler, camera_2: PinholeCameraCal3Bundler, point_3d: np.ndarray
+) -> float:
+    """Calculates the angle formed at the 3D point by the rays backprojected from 2 cameras.
```

Comment:
looks good. would a bit of ascii art help make the triangle relationship even clearer?

Reply:
thanks for the ascii art. much clearer!